action.yml

```diff
@@ -281,6 +281,14 @@ runs:
           echo "Running yarn lint"
           yarn run lint
         fi
+        # Validate licenses of all NPM dependencies are allowed
+        echo "Checking licenses of all dependencies"
+        # The following NPM package report as UNKNOWN or UNLICENSED, though have been manually checked they have permissive licenses:
+        EXCLUDE_PACKAGES='glob-to-regexp@0.3.0;jquery.are-you-sure@1.9.0;@silverstripe/react-injector@0.2.1;cwp-watea-theme@4.0.0;cwp-starter-theme@4.0.0'
```

Comment:
- `glob-to-regexp` is archived so we should replace it.
- `silverstripe/react-injector` isn't maintained - the correct react injector is inside `silverstripe/admin` directly. Whatever has this dependency should remove it immediately.
- `cwp-watea-theme` and `cwp-starter-theme` should not be included as npm/yarn dependencies _anywhere_, so whatever has listed those as dependencies should also remove them immediately.

`jquery.are-you-sure` is the only item here that we can't really do anything about.

Reply:
Yes we should sort those out for CMS 6, though remember this would also run on CMS 5.3 builds, and we don't have plans to update JS deps there

I've added a note to the [unmaintained deps](https://github.com/silverstripe/.github/issues/353) card to remove these

Reply:
I don't see any problem with resolving these dependency problems in CMS 5, or at the very very least looking at what effort is involved with doing so. I suspect the effort will be very small for these deps.

More to the point though, if we're not going to update deps for CMS 5 then this thing that checks deps shouldn't run on CMS 5.

Reply:
Replacing deps is out of scope for this PR. We can look CMS 5 deps seperately on the unmaintained deps card I linked to.

>  and we don't have plans to update JS deps there

I'm probably wrong there actually, we still haven't released CMS 5.4 which presumably will include updating JS deps.  We may also need to update deps while it's still under support post CMS 6 stable